### PR TITLE
Fix linter errors. Few warnings still need to be sorted out.

### PR DESCRIPTION
**Pull request by xmano**
I need help with few more lines of code..

libs\media\src\lib\file\viewers\utils.ts

```
(container.nativeElement as any).webkitRequestFullscreen();

(document as any).webkitExitFullscreen();

```

libs\media\src\lib\image\uploader\reference-path.pipe.ts
```
    const getRef = referencesPaths[type][key] as any;
    return getRef(docId)
```

libs\media\src\lib\file\explorer\explorer.component.ts  - can I remove ...args here?
```
  keepOrder(...args) { return 0; }

```

libs\media\src\lib\+state\file-uploader.service.ts
Promise.allSettled() .. configs to be changed by you.

### CODE REVIEW

File: libs/media/.eslintrc.json [hunk 1]
```diff
@@ -11,11 +11,11 @@
       "parserOptions": { "project": ["libs/media/tsconfig.*?.json"] },
       "rules": {
         "@angular-eslint/directive-selector": [
-          "warning",
+          1,
```
Comment (GrandSchtroumpf) [inline libs/media/.eslintrc.json:14]: It should be "warn" instead of 1

File: libs/media/src/lib/file/carousel/carousel.component.ts [hunk 2]
```diff
@@ -18,7 +18,7 @@ export class FileCarouselComponent {
   @Input() files: StorageFile[];
 
   /** Emit the file path that has been selected by the user */
-  @Output() select = new EventEmitter<string>();
+  @Output() selectedPath = new EventEmitter<string>();
```
Comment (GrandSchtroumpf) [inline libs/media/src/lib/file/carousel/carousel.component.ts:21]: Change to `selectPath`, make sure it's updated in every html file that use it (if any)

File: libs/media/src/lib/image/directives/imgix-helpers.ts [hunk 3]
```diff
@@ -53,7 +52,7 @@ export function getImgSize(ref?: string) {
 export function formatParameters(parameters: ImageParameters): string {
 
   const query = Object.entries(parameters)
-    .filter(([key, value]) => !!value)
+    .filter(([value]) => !!value)
```
Comment (GrandSchtroumpf) [inline libs/media/src/lib/image/directives/imgix-helpers.ts:55]: No, this is not at all the same thing. Change for that: 
```typescript
const query = Object.keys(parameters)
  .filter(key => parameters[key])
  .map(key => `${key}=${parameters[key]}`)
  .join('&');
```

File: libs/media/.eslintrc.json (hunk 1 above)
Comment (xmano) [reply libs/media/.eslintrc.json]: fixed

File: libs/media/src/lib/file/carousel/carousel.component.ts (hunk 2 above)
Comment (xmano) [reply libs/media/src/lib/file/carousel/carousel.component.ts:21]: Changed and verifying the clients of this component.

File: libs/media/.eslintrc.json (hunk 1 above)
Comment (xmano) [reply libs/media/.eslintrc.json]: done

File: libs/media/src/lib/image/directives/imgix-helpers.ts (hunk 3 above)
Comment (xmano) [reply libs/media/src/lib/image/directives/imgix-helpers.ts]: Fixed.

**Comment by xmano** [comment]
Seeing some build errors with the changes.. Have to fix those.

**Pull request merged**